test.js

```diff
@@ -1,17 +1,36 @@
 import test from 'ava';
 import sortKeys from '.';
 
+const orderedDeepEqual = (t, a, b) => {
+	t.deepEqual(a, b);
+	t.deepEqual(Object.keys(a), Object.keys(b));
+};
```

Comment:
`t.deepEqual` doesn't verify key order, which is critical in these tests.  So adding second assertion based on the keys.